asset-manager/README.md

```diff
@@ -181,72 +112,158 @@ scripts/start.sh
 
-Windows:
+### Install GitHub Copilot app modernization
 
-```batch
-scripts\deploy-to-azure.cmd -ResourceGroupName <your resource group name> -Location <your resource group location, e.g., eastus2> -Prefix <your unique resource prefix>
-```
+In VSCode, open the Extensions view from the Activity Bar, search for the `GitHub Copilot app modernization` extension in the marketplace. Click the Install button for the extension. After installation completes, you should see a notification in the bottom-right corner of VSCode confirming success.
 
-Linux:
+**Alternative: IntelliJ IDEA**
+Alternatively, you can use IntelliJ IDEA. Open **File** > **Settings** (or **IntelliJ IDEA** > **Preferences** on macOS), navigate to **Plugins** > **Marketplace**, search for `GitHub Copilot app modernization`, and click **Install**. Restart IntelliJ IDEA if prompted.
 
-```bash
-scripts/deploy-to-azure.sh -ResourceGroupName <your resource group name> -Location <your resource group location, e.g., eastus2> -Prefix <your unique resource prefix>
-```
+### Assess Your Java Application
+
+The first step is to assess the sample Java application `asset-manager`. The assessment provides insights into the application's readiness for migration to Azure.
+
+1. Open VS Code with all the prerequisites installed for the asset manager by changing the directory to the `asset-manager` directory and running `code .` in that directory.
+1. Open the `GitHub Copilot app modernization` extension.
+1. In the **QUICKSTART** view, click the **Migrate to Azure** button to trigger the Modernization Assessor.
+
+   ![Trigger Assessment](doc-media/trigger-assessment.png)
+
+1. Wait for the assessment to be completed and the report to be generated.
+1. Review the **Assessment Report**. Select the **Issues** tab to view the proposed solutions for the issues identified in the report.
+
+### Upgrade Your Java Application
+
+1. In the **Java Upgrade** table at the bottom of the **Issues** tab, click the **Run Task** button of the first entry **Java Version Upgrade**.
+
+    ![Java Upgrade](doc-media/java-upgrade.png)
+1. After clicking the **Run Task** button, the Copilot Chat panel will open with Agent Mode. The agent will check out a new branch and start upgrading the JDK version and Spring/Spring Boot framework. Click **Allow** for any requests from the agent.
+
+1. Refer to the [workshop/java-upgrade](https://github.com/Azure-Samples/java-migration-copilot-samples/tree/workshop/java-upgrade/asset-manager) branch for the upgraded status of the project. Additionally, if you have any problems with the Java upgrading step, you can start from that branch for the content below.
```

Comment:
I suggest we move this line to the next section, with some notes like

```
If starting freshly in this section, you can checkout xxx branch blabla
```

Reply:
If we switched the order of upgrade and assessment, it would make more sense to start from the updated branch, do assessment, and then migrate.